sql/views.sql

```diff
@@ -154,14 +154,17 @@ CREATE OR REPLACE VIEW timescaledb_information.continuous_aggregate_stats as
     cagg.job_id as job_id,
     bgw_job_stat.last_start as last_run_started_at,
     bgw_job_stat.last_successful_finish as last_successful_finish,
-    CASE when bgw_job_stat.last_run_success = 't' then 'Success'
-         when bgw_job_stat.last_run_success = 'f' then 'Failed'
-    END AS last_run_status,
-    case when bgw_job_stat.last_finish < '4714-11-24 00:00:00+00 BC' then 'running'
-       when bgw_job_stat.next_start is not null then 'scheduled'
-    end as job_status,
-    case when bgw_job_stat.last_finish > bgw_job_stat.last_start then (bgw_job_stat.last_finish - bgw_job_stat.last_start)
-    end as last_run_duration,
+    CASE WHEN bgw_job_stat.last_finish < '4714-11-24 00:00:00+00 BC' THEN NULL 
+         WHEN bgw_job_stat.last_finish is not null THEN
+              CASE WHEN bgw_job_stat.last_run_success = 't' THEN 'Success'
+                   WHEN bgw_job_stat.last_run_success = 'f' THEN 'Failed'
+               END
+    END as last_run_status,
+    CASE WHEN bgw_job_stat.last_finish < '4714-11-24 00:00:00+00 BC' THEN 'running'
+       WHEN bgw_job_stat.next_start is not null THEN 'scheduled'
```

Comment:
You are mixing the case of result columns. I would suggest to be consistent. Not sure if it is good time to change the text now since it might break APIs, but consider if you want to use Capitalized, lowercase, or UPPERCASE for all texts. We should also be consistent with other views.

(I think it is customary to use UPPERCASE for these kinds of enumeration-like constants, but I have no strong opinion on what we should use, except that we should try to do what users expect based on what other systems use and not invent our own convention.)

Reply:
modified.